ricty.rb

```diff
@@ -54,63 +60,49 @@ class Ricty < Formula
 
   patch do
     # workaround for #43 and #46
-    url "https://gist.githubusercontent.com/equal-l2/4c87ebea28f35846a47b596b9689b00e/raw/5a8a8df7e6d2fffa29127bd1a2ca0134a9745fc9/ricty.diff"
+    url "https://raw.githubusercontent.com/sanemat/homebrew-font/master/ricty_generator.patch"
```

Comment:
If I'll separately commit the patch to the repo beforehand, then I can use specific URL on this PR.
Is it OK?